scripts/deploy.py

```diff
@@ -79,6 +85,23 @@
         print("We've been waiting for an hour. That seems bad. Failing now.")
         sys.exit(1)
 
-    print("Looks good to release! Pushing the tag now.")
+    print("Looks good to release!")
+
+    if os.path.exists(DIST):
+        shutil.rmtree(DIST)
+
+    print("Now uploading to pypi.")
+
+    subprocess.check_output([
+        sys.executable, "setup.py", "sdist"
```

Comment:
Now we have an absolute path to `DIST`, let's make sure the `sdist` step puts it there:

```python
subprocess.check_output([
    sys.executable, "setup.py", "sdist", "--dist-dir", DIST
])
```